package.json

```diff
@@ -74,7 +79,6 @@
   "dependencies": {
     "@babel/core": "^7.21.0",
     "@babel/plugin-syntax-jsx": "^7.21.4",
-    "@babel/plugin-syntax-typescript": "^7.21.4",
     "@babel/types": "^7.21.3",
     "forgetti": "^0.4.2",
```

Comment:
Might wanna also bump `forgetti` here before releasing Next.js plugin. `forgetti` just fixed a bug a few hours ago: https://github.com/lxsmnsyc/forgetti/pull/4

Reply:
good to know